coursebuilder/modules/explorer/_static/components/top-bar/top-bar.html

```diff
@@ -29,7 +29,7 @@
                   <li><a href="#!tools"><i class="material-icons">work</i>Tools</a></li>
                   <li><a href="#!"><i class="material-icons">insert_invitation</i>Events</a></li>
                   <li><a href="#!partners"><i class="material-icons">people</i>Partners</a></li>
-                  <li><a href="#!faqs"><i class="material-icons">help</i>Help & FAQs</a></li>
+                  <li><a href="#!"><i class="material-icons">help</i>Help & FAQs</a></li>
```

Comment:
The href should point to '#!faqs' which the id of the div containing the page (the page itself is at `coursebuilder/modules/explorer/_static/components/pages/help-page.html`